doc/cephfs/fuse.rst

```diff
@@ -2,38 +2,60 @@
 Mount CephFS using FUSE
 =======================
 
-Before mounting Ceph via "Filesystem in USErspace" (FUSE), ensure that the client
-host has a copy of the Ceph configuration file and a keyring with CAPS for the
-Ceph metadata server.
+Pre-requisite
+=============
 
-#. From your client host, copy the Ceph configuration file from the monitor host 
-   to the ``/etc/ceph`` directory. :: 
+Before mounting Ceph via "Filesystem in USErspace" (FUSE), ensure that the
+client host (where CephFS has to mounted and used) has a copy of the Ceph
+configuration file (i.e. ``ceph.conf``) and a keyring of CephX user that has
+CAPS for the Ceph metadata server. Both of these files must be on the host
+where the Ceph monitor resides.
 
+#. Copy the Ceph conf from the monitor host to client host at
```

Comment:
Better, I think, to recommend they generate a minimal config with
```
ceph config generate-minimal
```

Reply:
It is better. I've made the change (hopefully, I got it right) - https://github.com/ceph/ceph/pull/30754/files#diff-1f08f2a244497174dd801aca5d275843R18.